test/0136/http-no-patch-delete.test.js

```diff
@@ -0,0 +1,54 @@
+const { linterForAepRule } = require('../utils');
+require('../matchers');
+
+let linter;
+
+beforeAll(async () => {
+  linter = await linterForAepRule('0136', 'aep-136-no-patch-delete');
+  return linter;
+});
+
+test('aep-136-no-patch-delete should find warnings', () => {
+  const oasDoc = {
+    openapi: '3.0.3',
+    paths: {
+      '/test1/{id}:archive': {
+        patch: {},
```

Comment:
To check for resilience to additionalProperties:
```suggestion
        patch: {},
        'x-custom': true,
```